### DISCUSSION TITLE
Status of development

### DISCUSSION THREAD

**Discussion by spo9kor** [nxrighthere/UnrealCLR#72] [General]
There are no major commits in the last 40 days. Is there any specific reason for this?
It is still showing .net 5 rc2. 
c#.net 5 is out now.

**Comment by schotime** [comment]
He's moving house.

**Reply by spo9kor** [reply]
ohh, okay.

**Reply by nxrighthere** [reply]
Yup. I migrated to another country and in-progress of getting a long-term residence permit, a lot of paperwork, and so on. The project is on hold at the moment.
